packages/cli/src/active-executions.ts

```diff
@@ -222,7 +227,7 @@ export class ActiveExecutions {
 	private getExecution(executionId: string): IExecutingWorkflowData {
 		const execution = this.activeExecutions[executionId];
 		if (!execution) {
-			throw new ExecutionNotFoundError(executionId);
+			ErrorReporterProxy.error(new ExecutionNotFoundError(executionId));
```

Comment:
removing the throw here changes the return type to `IExecutingWorkflowData | undefined`. We should either bring back the `throw`, or update the rest of the code to handle the `undefined`.

Reply:
🤦🏾 can't wait for enabling `noUncheckedIndexedAccess`...

I'm also in favor of the second. Let's see how much code it affects.